extensions/cli/geoserver/src/main/java/mil/nga/giat/geowave/cli/geoserver/GeoServerAddCoverageCommand.java

```diff
@@ -86,12 +94,8 @@ public void execute(
 				cvgName);
 
 		if (addLayerResponse.getStatus() == Status.OK.getStatusCode()) {
-			System.out.println("Add coverage '" + cvgName + "' to '" + workspace + "/" + cvgstore
-					+ "' on GeoServer: OK");
-		}
-		else {
-			System.err.println("Error adding GeoServer coverage " + cvgName + "; code = "
-					+ addLayerResponse.getStatus());
+			return "Add coverage '" + cvgName + "' to '" + workspace + "/" + cvgstore + "' on GeoServer: OK";
```

Comment:
this string response is kinda nice - more of our commands should be doing something like this.